src/server/api/routers/user.ts

```diff
@@ -49,6 +53,27 @@ export const userRouter = createTRPCRouter({
 
         const [firstUser, ...usersToDelete] = duplicateUsers;
 
+        // This section makes sure the logged in user does not get the achievement 201 twice.
+        if (firstUser) {
```

Comment:
`firstUser` is alway gonna be defined since we checked `duplicateUsers` length before

Reply:
Typescript still gives an error that it can be undefined.. that why the if statemenent

Reply:
You can type `firstUser!`